addons/keybinding/fnc_addKeybind.sqf

```diff
@@ -88,6 +88,10 @@ if (_defaultKeybind isEqualTypeParams [0, false, false, false]) then {
     _defaultKeybind = [_defaultKey, [_defaultShift, _defaultControl, _defaultAlt]];
 };
 
+// Handle empty strings to indicated no code
+if (_downCode isEqualType "") then {_downCode = {};};
+if (_upCode isEqualType "") then {_upCode = {};};
```

Comment:
The old method is consistent with the old way which only checked if it was a string
```
    if(IS_CODE(_downCode)) then {
        [_hashDown, "keydown"] call cba_fnc_removeKeyHandler;
```

Compiling could cause problems if they pass something like `"#nothing"`.
I figured it's deprecated anyway as there is no reason to not pass `{}`